### DISCUSSION TITLE
How to Force Google Maps to Render Off-Screen in React for Proper Page Printing?

### DISCUSSION THREAD

**Discussion by it-mukaseev** [visgl/react-google-maps#588] [Q&A]
I'm using React with Google Maps on a page that includes multiple sections. When I try to print the page, the map doesn’t render if it’s outside the viewport and requires scrolling down to become visible. Is there a way to force the map to render immediately, even if it’s below the fold, so that it displays properly for printing?

**Comment by alvinlimfuel** [comment]
Hi, did you find a solution?
